internal/dataparsers/blender_dataparser.py

```diff
@@ -65,17 +63,26 @@ def _parse_transforms_json(self, split: str) -> ImageSet:
         R = world_to_camera[:, :3, :3]
         T = world_to_camera[:, :3, 3]
 
+        height, width = None, None
+        for image_path in image_path_list:
+            img = cv2.imread(image_path)
+            if img is None:
+                continue
+            height, width = img.shape[:2]
+            break
+
         # parse focal length
         fx = torch.tensor(
             [fov2focal(fov=transforms["camera_angle_x"], pixels=width)],
             dtype=torch.float32,
         ).expand(R.shape[0])
-        # TODO: allow different fy
-        fy = torch.clone(fx)
+        fy = torch.tensor(
+            [fov2focal(fov=transforms["camera_angle_y"], pixels=height)],
```

Comment:
Have you tested it? Looks like the nerf synthetic dataset does not have `camera_angle_y`.

Reply:
@yzslab just checked and you are right. We have to use `camera_angle_x` only. So in theory it has same height and width.

Reply:
fixed.